cmd/create.go

```diff
@@ -171,9 +166,20 @@ A full list of component types that can be deployed is available using: 'odo com
 			fmt.Printf("Please wait, creating %s component ...\n", componentName)
 			err = component.Build(client, componentName, applicationName, false, true, stdout)
 			checkError(err, "")
-			fmt.Printf("Component '%s' was created.\n", componentName)
-			fmt.Printf("To push source code to the component run 'odo push'\n")
 		}
+
+		ports, err := component.GetComponentPorts(client, componentName, applicationName)
+		checkError(err, "")
+		fmt.Printf("Component '%s' was created", componentName)
+
+		if len(ports) > 1 {
+			fmt.Printf(" and ports %s were opened\n", strings.Join(ports, ","))
+		} else if len(ports) == 1 {
+			fmt.Printf(" and port %s was opened\n", ports[0])
+		}
+
+		fmt.Printf("To push source code to the component run 'odo push'\n")
```

Comment:
we don't need this line if our source is `git`

Reply:
Ack